test/daimon_skycrawlers/crawler_test.rb

```diff
@@ -3,14 +3,21 @@
 class DaimonSkycrawlersCrawlerTest < Test::Unit::TestCase
   sub_test_case 'fetch html' do
     setup do
-      @crawler = ::DaimonSkycrawlers::Crawler.new 'http://example.com'
+      stubs = Faraday::Adapter::Test::Stubs.new do |stub|
```

Comment:
Ah, right !
This is already specified at https://github.com/bm-sms/daimon-skycrawlers/pull/15/files#diff-3c3e0d9e34b3c93658c5b83e9dd85f53R11